### PR TITLE
Added a 'destroy engine' button to Vehicle Options.

### PR DESCRIPTION
**Pull request by NylanderrDK**
I received several requests for a 'Destroy Engine' button in the Vehicle Options submenu. So I made it and added it to the submenu. A preview of the feature can be seen here: https://streamable.com/saeycl

### CODE REVIEW

**Comment by Explooosion-code** [comment]
I would put the menu option before the delete option :D

**Comment by NylanderrDK** [comment]
> I would put the menu option before the delete option :D

That's definitely not a bad idea! Thank you.

File: vMenu/menus/VehicleOptions.cs [hunk 1]
```diff
@@ -497,6 +504,11 @@ private void CreateMenu()
                         {
                             SetVehicleEngineOn(vehicle.Handle, !vehicle.IsEngineRunning, false, true);
                         }
+                        // Destroy engine
+                        else if (item == destroyEngine)
+                        {
+                            SetVehicleEngineHealth(vehicle.Handle, -4000);
```
Comment (TomGrobbe) [inline vMenu/menus/VehicleOptions.cs:510]: Setting it to 0 should also kill it afaik, or maybe -1 should be good enough. I haven't tested it completely but let me know. Other than that this PR looks good.

**Comment by TomGrobbe** [comment]
Closing this PR because it's just been merged via PR #290

**Pull request closed without merging**